src/exec/Main.hs

```diff
@@ -222,6 +227,16 @@ cliParser = ByronProxyOptions
     Opt.metavar "[HOSTNAME]:SERVIVCENAME" <>
     Opt.help help
 
+  -- Uses the Read instance for Double.
+  cliPBftSignatureThreshold :: Opt.Parser PBftSignatureThreshold
+  cliPBftSignatureThreshold = Opt.option (PBftSignatureThreshold <$> Opt.auto) $
+    Opt.long "pbft-threshold" <>
+    Opt.metavar "DOUBLE" <>
+    -- FIXME ouroboros-consensus should export defaultPBftSignatureThreshold
+    -- That'd be better than putting this magic number here.
```

Comment:
Alternatively, I could make this an optional parameter with no default, and give `Nothing` to `protocolInfoByron`. Any strong arguments in either favour?

Reply:
You can make this a `Maybe` field, since `protocolInfoByron` accepts a maybe, and it uses the default if you pass nothing. That'd be the simplest.